### PR TITLE
Coremark: update to ver 2020-05-28 and fix for 64bit

### PR DESCRIPTION
**Pull request by kongfl888**
Maintainer: me
Compile tested: openwrt-master gita197fa0
Run tested: aarch64+openwrt-master gita197fa0

Description:
old version 2020-01-19 of coremark seems to have a problem, when platform is a aarch64. Update it.

### CODE REVIEW

**Comment by neheb** [comment]
Use

```
ifeq ($(CONFIG_ARCH_64BIT),y)
```

or something.

**Comment by kongfl888** [comment]
@neheb OK, thank you.

**Comment by kongfl888** [comment]
sorry for busyness.
aarch64:  test running successfully.

coremark-test20200824

@neheb thank you.

File: utils/coremark/Makefile [hunk 1]
```diff
@@ -8,13 +8,13 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=coremark
-PKG_SOURCE_DATE:=2020-01-19
-PKG_SOURCE_VERSION:=0c91314d1a4fdfc157d623ad5cb6ac5aef746db1
-PKG_RELEASE:=1
+PKG_SOURCE_DATE:=2020-05-28
+PKG_SOURCE_VERSION:=7685fd32bd7919581bfff2881a6dac6320581400
+PKG_RELEASE:=2
```
Comment (neheb) [inline utils/coremark/Makefile:13]: This does not need to be bumped.

**Comment by kongfl888** [comment]
@neheb like it?

**Pull request merged**